src/encoded/static/libs/svg-icons.js

```diff
@@ -40,3 +38,5 @@ var collapseIcon = module.exports.collapseIcon = function(collapsed, addClasses)
         </svg>
     );
 };
+
+module.exports.svgIcon = collapseIcon;
```

Comment:
Thanks for moving this file forward! We’re moving everything to ES6, so should do that for this file too. So `import React from 'react'` and `export const…` instead of `module.exports.…` would be great. The AirBnB eslint extension I think requires this, though I’m not sure. AirBnB itself does: https://github.com/airbnb/javascript#modules